src/trackers/DC.py

```
@@ -243,7 +243,8 @@ async def upload(self, meta, disctype):
         if not meta.get('debug', False):
             try:
                 upload_url = f'{self.api_base_url}/upload'
-                torrent_path = f"{meta['base_dir']}/tmp/{meta['uuid']}/BASE.torrent"
+                await self.common.create_torrent_for_upload(meta, self.tracker, 'DigitalCore.club')
+                torrent_path = f"{meta['base_dir']}/tmp/{meta['uuid']}/[{self.tracker}].torrent"
```

Ruff (0.14.10)

Probable insecure usage of temporary file or directory: "/tmp/"

(S108)